pkg/querier/worker/frontend_processor.go

```diff
@@ -58,7 +58,7 @@ func (fp *frontendProcessor) notifyShutdown(ctx context.Context, conn *grpc.Clie
 }
 
 // runOne loops, trying to establish a stream to the frontend to begin request processing.
-func (fp *frontendProcessor) processQueriesOnSingleStream(ctx context.Context, conn *grpc.ClientConn, address string) {
+func (fp *frontendProcessor) processQueriesOnSingleStream(ctx context.Context, conn *grpc.ClientConn, address, _ string) {
```

Comment:
is the `_` param intentional here?

Reply:
Yes, `processQueriesOnSingleStream` implements the `processor` interface and we don't use the worker ID in the case of the frontend processor.